src/editor/nodeDefs/io/strings.js

```diff
@@ -0,0 +1,158 @@
+          param3: {value:"", required: false}
+        },
+        inputs:1,   // set the number of inputs - only 0 or 1
+        outputs:1,  // set the number of outputs - 0 to n
+        faChar: "&#xf0cc;",  // 's' text icon
+        label: function() {
+            return this.name||"strings";
+        },
+        labelStyle: function() {
+            return this.name?"node_label_italic":"";
+        },
+
+        oneditprepare: function() {
+          var myFuncDef = stringFunctions[this.mode];
+
+
+          function handleMode(functionDef) {
+            $("#node-form-row-param2").hide();
+            $("#node-form-row-param3").hide();
+            functionDef.forEach(function(param) {
+              console.log("param", param);
+              if(param.param2) {
+                $("#node-label-param2").html(param.param2);
+                $("#node-form-row-param2").show();
+              }
+              if (param.param3) {   
+                $("#node-label-param3").html(param.param3);
+                $("#node-form-row-param3").show();
+              }
+            });
+          }
+
+          handleMode(myFuncDef);
+
+          var modeInput = $("#node-input-mode");
+          modeInput.change(function (){ 
+            console.log('modeInput changed', this.value);
+            handleMode(stringFunctions[this.value]);
+          })
+        },
+        render: function (){
+          return (
+            <div>
+              <div className="form-row">
+                <label htmlFor="node-input-mode"><i className="fa fa-gears"></i> <span>Func</span></label>
+                <select type="text" id="node-input-mode" style={{width:"74%"}}>
+                  <option value="camelCase">camelCase</option>
+                  <option value="capitalize">capitalize</option>
+                  <option value="deburr">deburr</option>
+                  <option value="endsWith">endsWith</option>
+                  <option value="escape">escape</option>
+                  <option value="escapeRegExp">escapeRegExp</option>
+                  <option value="kebabCase">kebabCase</option>
+                  <option value="lowerCase">lowerCase</option>
+                  <option value="lowerFirst">lowerFirst</option>
+                  <option value="pad">pad</option>
+                  <option value="padEnd">padEnd</option>
+                  <option value="padStart">padStart</option>
+                  <option value="parseInt">parseInt</option>
+                  <option value="repeat">repeat</option>
+                  <option value="replace">replace</option>
+                  <option value="snakeCase">snakeCase</option>
+                  <option value="split">split</option>
+                  <option value="startCase">startCase</option>
+                  <option value="startWith">startWith</option>
+                  <option value="toLower">toLower</option>
+                  <option value="toUpper">toUpper</option>
```

Comment:
toUpper is not working as intended for some reason when using a test case from Lodash documentation. https://lodash.com/docs#toUpper the same with